bip-0003.md

```diff
@@ -264,14 +266,16 @@ interfere as little as possible with ongoing adoption. If a Complete BIP is foun
 changes, it may be preferable to move it to Closed[^new-BIP], and to start a new BIP with the changes instead.
 Otherwise, it could cause confusion as to what being compliant with the BIP means.
 
-A BIP may remain in the Complete status indefinitely unless its authors decide to move it to Closed or it is advanced to
+A BIP may remain in the Complete status indefinitely unless its authors or deputies decide to move it to Closed or it is advanced to
 Deployed.
 Complete is the final status for most successful Informational BIPs.
 
 #### Deployed
 
-A settled[^settled] BIP may be advanced to Deployed upon request by any community member with evidence[^evidence] that the idea
-described in the BIP is in active use. Convincing evidence includes for example: an established project having deployed support
+Deployed BIPs should not be changed. A Complete BIP should only be moved to Deployed once it is settled: after its
```

Comment:
We may want to be clear what "changed" means here; perhaps something along the lines of:

s/not be changed/not be changed apart from minor fixes or editorial-only changes/